server/routes/bitbucket/bitbucket_selection.py

```
@@ -309,10 +312,15 @@ def cleanup_org_hooks(user_id: str, org_id: str, repo_full_names=None) -> list[s
             hooks = cur.fetchall()
 
     # Phase 2: Bitbucket API calls, no DB connection held.
+    # Same 3-wide pool as bulk enable — Bitbucket has no bulk webhook API.
+    def _one(item):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_one`

(ANN202)